### ISSUE TITLE
Transcoding vowel-marker characters

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/PWG#78]
In #76, @Andhrabharati proposes an extension to slp1 for representation of vowel-marker characters of Devanagari.  Let's continue that discussion here.

**Comment by funderburkjim** [comment]
In deva->slp1,  how do we know when to insert the marker? 
Also, how to type examples?  I use pramukhime to type Devanagari.

**Comment by Andhrabharati** [comment]
Does opening the new issue and 'continuing the discussion' mean that the point has been accepted in principle?
[otherwise, no point spending time on this!]

**Comment by funderburkjim** [comment]
>  point has been accepted in principle?

Not sure how to answer.  Since the need for such an extension is rare in the current CDSL,
this issue is not pressing.  However, I am curious to see whether the cdsl transcoding 
scheme can be extended in the way suggested.

**Comment by gasyoun** [comment]
> Since the need for such an extension is rare in the current CDSL, this issue is not pressing. 

agreee

**Comment by funderburkjim** [comment]
## a first test

Work appears in [issue78 directory](https://github.com/sanskrit-lexicon/PWG/tree/master/pwgissues/issue78).

There is no problem in extending slp1_deva1.xml to display the vowel-marker characters.
Compare test01_in.txt and test01_out.txt.

deva1_slp1.xml was extended in the most obvious way. Applying this to test01_out.txt  yields
test01_in_out.txt,  which is different from test01_in.txt.   
**Thus, invertibility fails** for this version of deva1_slp1.xml

Recall that a transcoding file specifies a finite-state machine, which processes a text string.
The deva1_slp1.xml state machine has only one state (named INIT).

Thus, it's finite state machine does not distinguish the context of, for instance, \u093e (Devanagari Vowel Sign A); so it always generates the slp1 'A'.

Note that slp1_deva1.xml takes 2 states to generate Devanagari from slp1.
Perhaps a version of deva1_slp1.xml with 2 states could properly distinguish the context.

**Comment by gasyoun** [comment]
> The deva1_slp1.xml state machine has only one state (named INIT).

Too abstract for me. For long time something I totally do not understand ))

**Comment by drdhaval2785** [comment]
JIm seems to have reported that he was able to create vowel marker characters in issue78 transcoder file. So this issue is closable.

**Comment by Andhrabharati** [comment]
No, he was saying he faced some invertibility issue which is failing!

But let this issue be closed, as the matter is not that serious (hardly a handful cases [< 10] exist in the entire CDSL repo)!
